integration/sql/iface-java/src/java/io/openlineage/sql/SqlMeta.java

```diff
@@ -0,0 +1,60 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.sql;
+
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class SqlMeta {
+    private final List<DbTableMeta> inTables;
+    private final List<DbTableMeta> outTables;
+
+    public SqlMeta(List<DbTableMeta> in, List<DbTableMeta> out) {
```

Comment:
will this abstraction suit well for column-lineage or is this out-of-scope of this PR ?
Asking this because we want table meta and column lineage meta in a single rust parser call and perhaps it makes sense to consider this although leaving column-lineage objects empty for now.

Reply:
This is out of scope for this PR, which just wants to achieve parity between Java and Python interfaces.
Separate PR will provide interface for column-level lineage.